packages/fx-core/src/plugins/solution/fx-solution/debug/scaffolding.ts

```diff
@@ -29,8 +29,6 @@ import { TelemetryEventName, TelemetryUtils } from "./util/telemetry";
 import { ScaffoldLocalDebugSettingsError } from "./error";
 import { isConfigUnifyEnabled } from "../../../../common/tools";
 
-const PackageJson = require("@npmcli/package-json");
```

Comment:
Shall we remove `@npmcli/package-json` from dependencies?

Reply:
Thx!